src/main/java/com/getindata/connectors/http/internal/table/sink/Slf4jHttpSinkPostRequestCallback.java

```diff
@@ -0,0 +1,30 @@
+package com.getindata.connectors.http.internal.table.sink;
+
+import java.net.http.HttpResponse;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+
+import lombok.extern.slf4j.Slf4j;
+
+import com.getindata.connectors.http.HttpSinkPostRequestCallback;
+import com.getindata.connectors.http.internal.sink.HttpSinkRequestEntry;
+
+@Slf4j
+public class Slf4jHttpSinkPostRequestCallback implements HttpSinkPostRequestCallback {
```

Comment:
Could you add a javadoc to this implementation

Reply:
added